content/1.7/back-office/international/locations/states-listing.md

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
Same here, this file should be in the "states" folder: content/1.7/back-office/international/locations/states/states-listing.md

Reply:
@Saimis777 I reopen this conversation because it's not fixed :)